src/listeners/ready.ts

```diff
@@ -1,11 +1,66 @@
-import { client } from "../index.js"
+import { client, Command } from "../index.js"
 import stats from "../events/stats.js"
 import inactives from "../events/inactives.js"
 import { listeningStatuses, watchingStatuses, playingStatuses } from "../config.json"
+import Discord from "discord.js"
 
 client.once("ready", async () => {
     console.log("Ready!")
 
+    const publishCommand = async (command: Command) => {
+        if (command.allowDM) {
+            //Create a global command
+            var cmd = await client.application?.commands.create({
+                name: command.name,
+                description: command.description,
+                defaultPermission: command.defaultPermission,
+                options: command.options
+            })
+        } else {
+            //Create a guild wide command
+            var cmd = await client.guilds.cache.get('841233609012543489')?.commands.create({
+                name: command.name,
+                description: command.description,
+                defaultPermission: command.defaultPermission,
+                options: command.options
+            }) 
+        }
+
+        let permissions: Discord.ApplicationCommandPermissionData[] = []
+        command.roleWhitelist?.forEach(id => { //Add whitelisted roles
```

Comment:
if there is a whitelist, the verified role should not have permission. Along with that, the everyone role should not have permission for any command except for verify

Reply:
I'll make it so it makes `defaultPermission` false if `roleWhitelist` exists.

Reply:
Alright, I wrote this:
```js
command.defaultPermission ? true : command.defaultPermission !== undefined ? false : command.roleWhitelist ? false : true
```
It looks absolutely horrible, so if you have a way of making it cleaner please do

Reply:
Can you explain what exactly you’re trying to do? I believe you could use `??` although I’m not 100% sure

Reply:
Yep, that works